ts/src/pro/test/Exchange/test.watchTrades.ts

```diff
@@ -1,77 +1,30 @@
-'use strict';
-
-// ----------------------------------------------------------------------------
 
 import assert from 'assert';
 import testTrade from '../../../test/Exchange/base/test.trade.js';
-import errors from '../../../base/errors.js';
-
-/*  ------------------------------------------------------------------------ */
-
-export default async (exchange, symbol) => {
-
-    // log (symbol.green, 'watching trades...')
+import testSharedMethods from '../../../test/Exchange/base/test.sharedMethods.js';
 
+async function testWatchTrades (exchange, skippedProperties, symbol) {
```

Comment:
@ttodua maybe we need to adjust the eslintrc?
![image](https://github.com/ccxt/ccxt/assets/43336371/4e79bfb0-7af1-402f-a6df-f8269f7cfabc)

Reply:
will fix this soon, found the reason

Reply:
ok, lmk me too what's the reason (i don't see those eslint warning, even though we interact with same eslintrc)

Reply:
you sure your eslint is working properly? can you run npm i again? we're missing a rule to disbale the jsdocs thing